### ISSUE TITLE
How to get the weights

### ISSUE THREAD

**Issue by ankitpatnala** [tensorflow/graphics#120]
I used mesh_segmentation_demo.py code of yours on my dataset. I saved all the checkpoints but how can i get the weights of the synapses matrix ?

**Comment by julienvalentin** [comment]
@avneesh-sud and @amakadia should be able to assist.

Best.

**Comment by ankitpatnala** [comment]
Is this feastNet algorithm translation , rotational and scalability invariant ? Paper had information about translation invariant but not the other two !

**Comment by avneesh-sud** [comment]
Hi @ankitpatnala 

The mesh-segmentation-demo colab notebook demonstrates how to load saved checkpoints from a trained model (see under 'Test model & visualize results').

FeastNet using extrinsic coordinates is not rotation or scalability invariant.

**Comment by ankitpatnala** [comment]
Hi @avneesh-sud ,
I am quite new to software and machine learning world. I have some silly queries.
The architecture in mesh-segmentation-task. Does it have any FCN ? i could only see the tf.keras.CONV1D. Why was no dropout choosed ? Do we have softmax activation function at the output layer ?
My problem:
I am currently working with some models which has varied number of vertices in a mesh.it was working well with small simple use cases. But now after increasing complexity the architecture failed to detect any vertex . It showed each and every vertex as class 0. The other change was i increased the number of input dataset to 4800 from 500.
